docs/_component/editor.client.js

```diff
@@ -163,6 +193,41 @@ export function Editor({children}) {
         </TabPanel>
         <TabPanel>
           <form className="frame-body frame-body-box frame-body-box-fixed-height">
+            <fieldset>
+              <label>
+                <input
+                  type="radio"
+                  name="language"
+                  checked={!state.formatMd}
+                  onChange={() => {
+                    setConfig({...state, formatMd: false})
+                  }}
+                />{' '}
+                Use <code>MDX</code>
+              </label>
+              <span style={{margin: '1em'}}>{' | '}</span>
```

Comment:
```suggestion
              <span style={{margin: '1em'}}>{' '}</span>
```

hmm I think the `|` looks a bit weird, but I don’t want to spend too much time on the design now and don’t care too much.
So this is optional. I want to improve the site some more soon anyway!